index.html

```diff
@@ -12,168 +12,171 @@
 </head>
 
 <body>
-
-
-
-<div class="page_wrapper">
-
-
-  <header class="position_rel">
-    <div class="position_fix green">
-  <div class="section_wrapper section header flex_container">
-    <div class="logo">ReMotely</div>
-    <nav class="nav flex_container">
-      <div class="link_flex_container"><a class="links" href="#">book</a><a class="links" href="#">explore</a>
-        <a class="links" href="#">about</a>
-      </div>
-      <div class="sign_container"><a class="sign" href="#">sign in</a></div>
-    </nav>
-  </div></div>
-</header>
-
-<section class="green_section">
-  <div class=" section_hero">
-    <div class="section_container">
-      <div class="column phrase">
-        <div class="catch_phrase">
-          <h1 class="invite_phrase">Work from <br> a cabin near Helsinki</h1>
-          <div class=button_container>
-            <button class="btn" type="submit">Explore</button>
-          </div>
+  <div class="page_wrapper">
+    <header class="header">
+      <div class="header_wrapper">
+        <div class="section_wrapper section header flex_container">
+          <h2 class="logo">
+            ReMotely
+          </h2>
+          <nav class="nav flex_container">
+            <div class="link_flex_container">
+              <a class="links" href="#">book</a>
+              <a class="links" href="#">explore</a>
+              <a class="links" href="#">about</a>
+            </div>
+            <div class="sign_container">
+              <a class="sign" href="#">sign in</a>
+            </div>
+          </nav>
         </div>
       </div>
-      <div class="column cottage">
-        <div class="photo">
-          <img class="main_photo" src="switzphoto.jpg" alt="chatka ze Szwajcarii">
-        </div>
-      </div>
-    </div>
-    
-  </div>
-</section>
- 
-  <section class="section second_page_container">
-    <h2 class="section_title">We offer a handpicked range of cabins for nature lovers in Europe to live and work
-      from for desired time.</h2>
-    <div class="all_pros_container">
-
-      <div class="pros_container">
-        <div class="photo_description_container">
-          <div class="pros_photo"><img class="pros_pic" src="forest.png" alt="zdjęcie zalet"></div>
-
-          <div class="header_container">
-            <h3 class="paragraph_header">Connection</h3>
+    </header>
+    <section class="green_section">
+      <div class=" section_hero">
+        <div class="section_container">
+          <div class="column phrase">
+            <div class="catch_phrase">
+              <h1 class="invite_phrase">
+                Work from <br> a cabin near Helsinki
```

Comment:
remove the `<br>` tag

Reply:
fixed finally but am not sure if that displays properly